### PR TITLE
ft(about_us): designed the about us page for the ecommerce app

### PR DESCRIPTION
**Pull request by Pacific911**
#### What does this PR do?

- Shows more about the e commerce website 
-  what the services they offer 
- Places and you can find them
- 
#### Description of Task to be completed?
- /about_us

#### Screenshots (if appropriate)

screen shot

### CODE REVIEW

File: src/pages/accounts/Accounts.tsx [hunk 1]
```diff
@@ -3,6 +3,7 @@ import { Route, Routes } from 'react-router-dom';
 import { AccountsLoader } from '../../components/Loaders';
 
 const Tfa = lazy(() => import('./tfa/Tfa'));
+// const signup = lazy(() => import('.'));
```
Comment (princoo) [inline src/pages/accounts/Accounts.tsx:7]: clean this line

Comment (Pacific911) [reply src/pages/accounts/Accounts.tsx]: solved 👍

**Review by munezerobagira — APPROVED** [review]

**Review by AdrineUWERA — APPROVED** [review]

**Review by AdrineUWERA — APPROVED** [review]

File: src/pages/about/about_us.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  imageSrc: string;
+}
+
+const teamMembers: TeamMember[] = [
+  { name: 'Norbert ISHIMWE', imageSrc: teamImages[0] },
+  { name: 'Pacific NDUWUMWE', imageSrc: teamImages[1] },
+  { name: 'Eic NSANZIMFURA', imageSrc: teamImages[3] },
+  { name: 'Edddy MUGISHA', imageSrc: teamImages[4] },
+  { name: 'Joseph NTWALI', imageSrc: teamImages[5] },
+  { name: 'Said MIRINDI', imageSrc: teamImages[6] },
```
Comment (Edcreation) [inline src/pages/about/about_us.tsx:16]: Name

**Review by princoo — APPROVED** [review]

**Review by Edcreation — CHANGES_REQUESTED** [review]

**Review by JosephNgabo — APPROVED** [review]

File: src/pages/about/about_us.tsx (hunk 2 above)
Comment (Pacific911) [reply src/pages/about/about_us.tsx:16]: requested changes made

**Review by Edcreation — APPROVED** [review]

**Pull request merged**